packages/react/src/components/FileUploader/index.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { boolean, withKnobs } from "@storybook/addon-knobs";
+
+import FileUploader from ".";
+
+const inputName = "input-name";
+const onChange = console.log;
```

Comment:
Ugo for tracking events you can use knob `actions` so that they actually get logged in Storybook, not in browser console:
https://github.com/heetch/flamingo/blob/master/packages/react/src/components/ListItem/index.stories.js#L78

Reply:
Oh nice, done!